browser/resources/extensions/brave_item_list_more_items.ts

```diff
@@ -1,54 +0,0 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-import { CrLitElement } from '//resources/lit/v3_0/lit.rollup.js'
```

Comment:
This component (and friends) wasn't used anywhere. I imagine a patch stopped applying at some point